browse-scaler/localstack/setup-localstack.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/sh
```

Comment:
Add a section to the README to explain how to test things out with Localstack.

Reply:
I will do this when I open a localstack PR